Developer Samples/CommandLineTools/BuildIrJsRuleApp/Program.cs

```diff
@@ -101,6 +101,15 @@ static async Task<int> Main(string[] args)
             if(ruleApp == null)
             {
                 Console.WriteLine("You must provide either RuleAppPath or all of CatUri, CatUsername, CatPassword, and CatRuleAppName (with optional CatLabel)");
+                return 1;
```

Comment:
Have you considered using `Environment.Exit(1)` instead of `return 1`? Another potential approach would be to set `Environment.ExitCode = 1; return;`? This might allow you to enrich output for errors before exiting the process (if needed)

Reply:
Given that this is in the context of a sample that someone could rebuild into their own application, I think I prefer making it a more generic "return" rather than something explicit to a console application - but a good suggestion!